composer.json

```diff
@@ -12,7 +12,7 @@
   },
   "require": {
     "php": "^8.0",
-    "pimcore/pimcore": "^10.2",
+    "pimcore/pimcore": "^10.5",
```

Comment:
Maybe we should check if the getLanguages method is available or not and exclude/include it from/to the schema?

Reply:
Just saw the other pr, nevermind, since we have to bump it to 10.5 anyways (because of the plugins)